pypeit/core/tracewave.py

```diff
@@ -555,6 +555,12 @@ def trace_tilts(arcimg, lines_spec, lines_spat, thismask, slit_cen, inmask=None,
                                    tcrude_maxshift0=tcrude_maxshift0, tcrude_nave=tcrude_nave,
                                    show_tracefits=show_tracefits)
 
+    # DP: Added this because sometime there are < 2 usable arc lines for tilt tracing, PCA fit does not work
```

Comment:
I think the logic would be cleaner if you make this an if/then statement right before line 579 below where use is calculated. i.e. if nuse > 2 do the PCA, if not return the trace_dict0.

Reply:
Done